### PR TITLE
[ATLAS-12985] Added workaround that allows to work with DropDown item as array always.

### PR DESCRIPTION
**Pull request by andrei-maroz**

### CODE REVIEW

File: main/mainTemplate.json [hunk 1]
```diff
@@ -64,12 +64,12 @@
...
-      "type": "string",
-      "defaultValue": "false",
+    "securityGroup": {
+      "type": "array",
+      "defaultValue": "bootstrapUi",
       "allowedValues": [
-        "false",
-        "true"
+        "bootstrapUi",
+        "ssh"
```
Comment (nnallamo) [inline main/mainTemplate.json:72]: we are not allowing SSH port. do we need to mention in allowed values ?

Comment (jkraj) [reply main/mainTemplate.json:72]: Then we don't add ssh?

Comment (nnallamo) [reply main/mainTemplate.json:72]: yes not required

**Review by joytandon17 — APPROVED** [review]

File: main/mainTemplate.json (hunk 1 above)
Comment (andrei-maroz) [reply main/mainTemplate.json:72]: Hi @nnallamo ,
This is just for QA version image.

**Pull request merged**